src/style.css

```
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
```

Use only class selector, fix it everywhere
```suggestion
.header {
```